server/services/project_service.py

```diff
@@ -303,7 +300,8 @@ def evaluate_validation_permission(
                 for team_dto in teams_allowed
                 if TeamService.is_user_member_of_team(team_dto.team_id, user_id)
             ]
-            if len(user_membership) == 0:
+            # Check for validator role for initial roll out
+            if not (len(user_membership) or UserService.is_user_validator(user_id)):
```

Comment:
This logic is wrong. If `len(user_membership)` is positive and the user is validator it would return the MappingNotAllowed exception.

Furthermore, we can not support the `is_user_validator` any more, as it would make TM not follow the teams restriction.

Reply:
Furthermore the frontend is not allowing a validator role user to make that request, so it would not be useful.